### PR TITLE
[ConstitutiveLawsApp] Update of the plastic-damage model

### PR DESCRIPTION
**Pull request by AlejandroCornejo**
**📝 Description**
An update of the associative plastic-damage model, now fully consistent according to plastic-damage dissipation.

Test is also updated.
Rankine plastic potential added.

### CODE REVIEW

File: applications/ConstitutiveLawsApplication/custom_constitutive/associative_plastic_damage_model.cpp [hunk 1]
```diff
@@ -95,7 +97,7 @@ void AssociativePlasticDamageModel<TYieldSurfaceType>::CalculateMaterialResponse
 
         plastic_damage_parameters.NonLinearIndicator = plastic_damage_parameters.UniaxialStress - mThreshold;
 
-        if (plastic_damage_parameters.NonLinearIndicator <= std::abs(1.0e-8 * mThreshold)) {
+        if (plastic_damage_parameters.NonLinearIndicator <= std::abs(tolerance * mThreshold)) {
```
Comment (loumalouomega) [inline applications/ConstitutiveLawsApplication/custom_constitutive/associative_plastic_damage_model.cpp:100]: At last!

**Review by loumalouomega — DISMISSED** [review]
Okay from my side

File: applications/ConstitutiveLawsApplication/custom_constitutive/associative_plastic_damage_model.cpp (hunk 1 above)
Comment (AlejandroCornejo) [reply applications/ConstitutiveLawsApplication/custom_constitutive/associative_plastic_damage_model.cpp:100]: :D

**Comment by AlejandroCornejo** [comment]
we need an aproval from the @KratosMultiphysics/technical-committee because of this spurious change in the strategy (there is no change)

**Review by SergioJimenezReyes — APPROVED** [review]
It is ok from my side

**Pull request merged**